config/development_sqlite.yaml

```diff
@@ -0,0 +1,112 @@
+log:
+  stdout: true
+  level: error
+
+persistence:
+  defaultStore: default
+  visibilityStore: visibility
+  numHistoryShards: 1
+  datastores:
+    default:
+      sql:
+        user: ""
+        password: ""
+        pluginName: "sqlite"
+        databaseName: "default"
+        connectAddr: "localhost"
+        connectProtocol: "tcp"
+        connectAttributes:
+          mode: "memory"
+          cache: "private"
+          setup: true
```

Comment:
You don't need this `setup` parameter anymore, right?

Reply:
Yes, it's no longer needed.